### PR TITLE
Fix the getResources never returning anything bug (thanks gastaldi)

### PR DESCRIPTION
**Pull request by jsight**

### CODE REVIEW

File: utils/src/main/java/org/jboss/windup/util/FurnaceCompositeClassLoader.java [hunk 1]
```diff
@@ -83,11 +83,12 @@ public URL getResource(String name)
         ClassLoader contextLoader = Thread.currentThread().getContextClassLoader();
         if (contextLoader != null)
         {
-            return contextLoader.getResources(name);
+            result.addAll(Collections.list(contextLoader.getResources(name)));
```
Comment (gastaldi) [inline utils/src/main/java/org/jboss/windup/util/FurnaceCompositeClassLoader.java]: result should be a LinkedHashSet. It doesn't seem valid to have two identical resources in it. Same goes for the list of classloaders in this class

Comment (jsight) [inline utils/src/main/java/org/jboss/windup/util/FurnaceCompositeClassLoader.java]: Good point.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/661/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/31/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/662/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/32/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/663/
Test PASSed.

**Pull request merged**